sdk/core/azure-core-http-vertx/src/main/java/com/azure/core/http/vertx/implementation/VertxRequestWriteSubscriber.java

```diff
@@ -122,19 +130,50 @@ private void onErrorInternal(Throwable throwable) {
         // code 2 and greater are completion states which means the error should be dropped as we already completed.
         if (state.code >= 2) {
             Operators.onErrorDropped(throwable, Context.of(contextView));
```

Comment:
do we need to call it at all from the logic perspective? I'd suggest to log it ourselves and maybe as a warning (unless we know it's severe issue)

Reply:
Same question where I wanted to respect reactive flow